frontend/device/tolino/device.lua

```diff
@@ -0,0 +1,33 @@
+local logger = require("logger")
+local A, android = pcall(require, "android")  -- luacheck: ignore
+local AndroidDevice = require("device/android/device")
+
+local function yes() return true end
+local function no() return false end
+
+local TolinoDevice = AndroidDevice:new{
+    model = "Tolino"
+}
+
+function TolinoDevice:init()
+    AndroidDevice.init(self)
+    self.input.event_map = require("device/tolino/event_map")
+end
+
+-- Tolino Vision 5, hardware model "E70K00"
+local TolinoE70K00 = TolinoDevice:new{
+    model = "Tolino_E70K00",
+    hasEinkScreen = yes,
```

Comment:
I'm afraid this doesn't work that way.

Here you're saying: hey I'm a eink screen. Let me refresh myself, but you didn't specify the driver kind, the type of waveforms you support....

With that option alone you won't get proper refreshes. Only a handful of messages on the log buffer saying that FakeEPDController issued some updates.